### DISCUSSION TITLE
Poll: Make Ctrl+Backspace consistent with other applications

### DISCUSSION THREAD

**Discussion by Broxzier** [OpenRCT2/OpenRCT2#19468] [Polls]
In almost all applications where you can type text, `Ctrl`+`Backspace` removes the text before the cursor indicator up to the start of the word. In OpenRCT2, however, is removes all text instead. I'd like to change this behaviour to be consistent.

As someone who frequently uses `Ctrl`+`Backspace` (and `Ctrl`+`Delete`) to remove entire words at once, it's annoying when you do this in OpenRCT2 and end up removing everything.

On the other hand, some players may be used to pressing `Ctrl`+`Backspace` to clear a text field entirely. With this changed, they may have to press it a few times instead. I believe this minor inconvenience doesn't outweighs the inconvenience of accidentally removing everything.

**Comment by Joris-Driessen** [comment]
Yes, this issue has been bothering me some time. I think having consistent behaviour over what users expect is quite important. By making it consistent with the behaviour in Windows, I think a lot of headache will be spared.
